packages/http/src/mocker/HttpMocker.ts

```diff
@@ -79,15 +79,14 @@ function isINodeExample(nodeExample: INodeExample | INodeExternalExample | undef
 
 function computeMockedHeaders(headers: IHttpHeaderParam[], ex: PayloadGenerator): Promise<Dictionary<string>> {
   const headerWithPromiseValues = mapValues(keyBy(headers, h => h.name), async header => {
-    if (header.content) {
```

Comment:
Out of interest, why did we flattened it?

Reply:
There has been kind of a long discussion with @marbemac, who was kind of concerned of the size of the operation created by graphite.

Basically his rationale was — instead of having a lot of empty things that effectively take space, let's set them as possibly undefined.